app/estimate_square_size_test.go

```diff
@@ -148,37 +148,44 @@ func Test_calculateCompactShareCount(t *testing.T) {
 		wPFDCount, messgeSize int
 	}
 	tests := []test{
-		{"empty block minimum square size", 0, 0, 0},
-		{"full block with only txs", 10000, 0, 0},
-		{"random small block square size 4", 0, 1, appconsts.SparseShareContentSize * 2},
-		{"random small block square size 8", 0, 1, appconsts.SparseShareContentSize * 4},
-		{"random small block w/ 10 normal txs square size 4", 10, 1, appconsts.SparseShareContentSize * 8},
-		{"random small block square size 16", 0, 4, appconsts.SparseShareContentSize * 8},
-		{"random medium block square size 32", 0, 50, appconsts.SparseShareContentSize * 8},
-		{"full block max square size", 0, 8000, appconsts.SparseShareContentSize / 2},
-		{"overly full block", 0, 80, appconsts.SparseShareContentSize * 100},
-		{"one over the perfect estimation edge case", 10, 1, appconsts.SparseShareContentSize + 1},
+		{"empty block minimum square size", 0, 0, totalMsgSize(0)},
+		{"full block with only txs", 10000, 0, totalMsgSize(0)},
+		{"random small block square size 4", 0, 1, totalMsgSize(appconsts.SparseShareContentSize * 2)},
+		{"random small block square size 8", 0, 1, (appconsts.SparseShareContentSize * 4)},
+		{"random small block w/ 10 normal txs square size 4", 10, 1, totalMsgSize(appconsts.SparseShareContentSize * 8)},
+		{"random small block square size 16", 0, 4, totalMsgSize(appconsts.SparseShareContentSize * 8)},
+		{"random medium block square size 32", 0, 50, totalMsgSize(appconsts.SparseShareContentSize * 8)},
+		{"full block max square size", 0, 8000, totalMsgSize(appconsts.SparseShareContentSize / 2)},
+		{"overly full block", 0, 80, totalMsgSize(appconsts.SparseShareContentSize * 100)},
+		{"one over the perfect estimation edge case", 10, 1, totalMsgSize(appconsts.SparseShareContentSize + 1)},
 	}
 	encConf := encoding.MakeConfig(ModuleEncodingRegisters...)
 	signer := generateKeyringSigner(t, "estimate-key")
 	for _, tt := range tests {
-		txs := generateManyRawWirePFD(t, encConf.TxConfig, signer, tt.wPFDCount, tt.messgeSize)
-		txs = append(txs, generateManyRawSendTxs(t, encConf.TxConfig, signer, tt.normalTxs)...)
-
-		parsedTxs := parseTxs(encConf.TxConfig, txs)
-		squareSize, totalSharesUsed := estimateSquareSize(parsedTxs, core.EvidenceList{})
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
[for reviewers] this code diff looks scary but everything in this block was indented by one level because of the addition of this line.

Note, we want `t.Run(tt.name, func(t *testing.T) {` so that we can easily identify which test failed by name in test output